staging/src/k8s.io/apiserver/plugin/pkg/authenticator/token/oidc/oidc.go

```diff
@@ -323,6 +329,25 @@ func (a *Authenticator) AuthenticateToken(token string) (user.Info, bool, error)
 			info.Groups[i] = a.groupsPrefix + group
 		}
 	}
+
+	// check to ensure all required claims are present in the ID token and have matching values.
+	if len(a.requiredClaims) != 0 {
```

Comment:
nit: if statement is redundant. Ranging over a nil map is already a no-op https://play.golang.org/p/joqoexhvvbN

Reply:
Done